app.js

```diff
@@ -5,17 +5,18 @@ const {
   handleCustomErrors,
   handlePsqlErrors,
 } = require("./errors/errors");
+const { getApiEndpoints } = require("./controllers/endpoints.controllers");
 const app = express();
-app.use(express.json());
 
+app.get("/api", getApiEndpoints);
 app.get("/api/topics", getTopics);
 
-app.use(handleCustomErrors);
-app.use(handlePsqlErrors);
-app.use(handleServerErrors);
-
 app.all("*", (req, res) => {
   res.status(404).send({ msg: "Route not found" });
 });
 
+app.use(handleCustomErrors);
+app.use(handlePsqlErrors);
+app.use(handleServerErrors);
+
```

Comment:
These shouldnt be added for this endpoint and have not been tested so should be removed and only added when needed/tested for.